actors/runtime/src/main/java/cloud/orbit/actors/Stage.java

```diff
@@ -136,7 +136,7 @@
 {
     private Logger logger = LoggerFactory.getLogger(Stage.class);
 
-    private static final int DEFAULT_EXECUTION_POOL_SIZE = 128;
+    private static final int DEFAULT_EXECUTION_POOL_SIZE = 32;
```

Comment:
If someone needs 128 threads they should probably be setting that when creating the Stage..